docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
-For example, you can create a silence to suppress trivial notifications during weekends.
+2. Custom templates created or uploaded on the **Alerting page > Alert Templates** tab.
+3. Custom template files available in your ``srv/alerting/templates`` directory. PMM loads them during startup.
 
 ### Contact points
-Contact points specify how PMM should deliver Grafana-managed alerts. When an alert fires, a notification is sent to the specified contact points. 
 
-Depending on the severity of an alert, you might want to send different alerts to different channels. For example, you can deliver common alerts via Slack channel, but send an email notification for potentially critical issues. 
+Contact points specify how PMM should deliver Grafana-managed alerts. When an alert fires, PMM sends a notification to the specified contact points.
+
+Depending on the severity of an alert, you might want to send different alerts to different channels. For example, you can deliver common alerts via Slack channel, but send an email notification for potentially critical issues.  
+
+PMM uses Email as the default contact point but you can choose from a variety of contact points, including Slack, Email, Webhooks, PagerDuty, and more.
+
+Before PMM can send out email notifications via Email contact points, you will need to:
+
+1. Configure Email (SMTP) server settings to add your email account settings
+2. Configure a contact point to define the email delivery options
+
+Contact points with invalid settings show a **No Attempts** status under {{icon.bell}}  **Alerting > Contact points**.
+
+### Configure Email (SMTP) server settings
+
+To set up email notifications, edit the PMM’s **grafana.ini** file. This holds the settings for [Grafana’s Alertmanager](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/alertmanager/) which routes notifications to contact points.
 
-You can choose from a variety of contact points, including Slack, email, webhooks, PagerDuty, and more. 
+To use SMTP with a PMM Docker installation:
+
+1. Create an `.env` file and populate it with your SMTP credentials (and other environment variables) as follows:
+
+        ```
+        GF_SMTP_ENABLED=true  
+        GF_SMTP_HOST=smtp.gmail.com:587
+        GF_SMTP_USER=email@domain.com
+        GF_SMTP_PASSWORD=<YOUR_SMTP_PASSWORD>
+        GF_SMTP_SKIP_VERIFY=false
+        GF_SMTP_FROM_ADDRESS=email@domain.com
+        GF_SMTP_FROM_NAME=Percona Alerting
+        ```
+
+2. Pass in the `.env` file to Docker run using the `--env-file` flag: ```docker run --env-file=.env -p 443:443 -p 80:80 percona/pmm-server:2```.
+
+### Restore SMTP settings following an upgrade
+
+Upgrading PMM will overwrite the **grafana.ini** file with a newer version. If you configured PMM to use SMTP settings via environment variables, you do not need to do anything after an upgrade as your settings will be transferred.
```

Comment:
I believe this ONLY happens if PMM using Docker - I believe this should be explicitly stated.
Change 'configured' to configure
Change 'will be transferred' to settings will be intact from the previous version (pre-upgraded)"